File: serv/src/main/java/com/example/serv/models/AddressDataModel.java
package com.example.serv.models;

import lombok.AllArgsConstructor;

import java.io.Serializable;

@AllArgsConstructor
public class AddressDataModel implements Serializable {

    public int idUser;
    public String street;
    public String home;
    public String flat;
    public String comment;
    public String homeName;

}

File: app/src/main/java/com/example/lr2/datamodel/AddressDataModel.kt
package com.example.lr2.datamodel

/***
 * Класс с данными для адреса пользователя
 */
data class AddressDataModel (
    var idUsed : Int,
    var street : String,
    var home : String,
    var flat : String,
    var homeName : String
) : java.io.Serializable